### DISCUSSION TITLE
can't /usr/bin/minio

### DISCUSSION THREAD

**Discussion by Elvis-ch1** [minio/minio#13229] [Q&A] [answered]
Hi we were trying to upgrade to minio RELEASE.2021-09-15T04-54-25Z, but it seems this path (/usr/bin/minio) has been removed from the minio docker image, build keeps failing at the stage. please whats the path for the minio executable binary in the minio docker image?

**Comment by klauspost** [comment, marked as answer]
`/opt/bin/minio`, but you shouldn't need to use direct paths. `minio` is in path, and you shouldn't need full path.

**Reply by Elvis-ch1** [reply]
okay thank you, did this recently change in minio RELEASE.2021-09-15T04-54-25Z? because the previous used to build just fine 
pics attached.

![minio1](https://user-images.githubusercontent.com/85713580/133658307-8614d015-e713-4f98-8a0c-4820b3bf28e5.png)
this was the build for minio RELEASE.2021-09-09T21-37-07Z

![minio2](https://user-images.githubusercontent.com/85713580/133658455-e917630a-a102-4aa9-ac8c-dac8403c46f1.png)
and this is for minio RELEASE.2021-09-15T04-54-25Z

**Reply by harshavardhana** [reply]
Yes it has changed @Elvis-ch1

**Reply by Elvis-ch1** [reply]
okay, many thanks for the prompt response
